app/policies/job_posting_policy.rb

```diff
@@ -0,0 +1,26 @@
+class JobPostingPolicy < ApplicationPolicy
+  def new?
+    @user.company?
```

Comment:
We talked about this offline, it's a good idea to also allow super admin to create job postings

Reply:
Will implement in a follow PR.